src/Common/Internal/Http/HttpClient.php

```diff
@@ -56,24 +58,31 @@ class HttpClient implements IHttpClient
 
     /**
      * Holds the latest response object.
-     * 
+     *
      * @var \HTTP_Request2_Response
      */
     private $_response;
 
     /**
      * Holds expected status code after sending the request.
-     * 
+     *
      * @var array
      */
     private $_expectedStatusCodes;
 
+    static function initProxy()
+    {
+        // read HTTP_PROXY enviroment variable, if any
```

Comment:
Please add an example of the HTTP_PROXY variable and also indicates this can be used for fiddler monitoring